CvGameCoreDLL_Expansion2/CvMap.cpp

```diff
@@ -1152,6 +1152,30 @@ CvArea* CvMap::findBiggestArea(bool bWater)
 }
 
 
+//	--------------------------------------------------------------------------------
+CvLandmass* CvMap::findBiggestLandmass(bool bWater)
```

Comment:
I'm just wondering if return can be cached? 
I'm not sure how this process works but if map is already generated is there a reason to rerun following code when it is called in lua? Maybe this should rerun only if new generation process start?